mmcv/ops/nms.py

```diff
@@ -33,38 +33,6 @@ def forward(ctx: Any, bboxes: Tensor, scores: Tensor, iou_threshold: float,
             inds = valid_inds[inds]
         return inds
 
-    @staticmethod
-    def symbolic(g, bboxes, scores, iou_threshold, offset, score_threshold,
```

Comment:
not sure will this have any side effect on parrots @zhouzaida

Reply:
Some users may use mmcv and parrots.convert to convert model with these symbolic fucntions in ext op and we can solve it by wrappers and examples in parrots.convert. So it does not matter.